packages/eslint-plugin-template/src/utils/is-on-same-location.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  TmplAstBoundAttribute,
+  TmplAstBoundEvent,
+} from '@angular-eslint/bundled-angular-compiler';
+
+export function isOnSameLocation(
```

Comment:
How come this utility was singled out to be promoted to its own file?

Reply:
Can't recall a reason - I'll move it to `attributes-order.ts`